modules/ROOT/nav.adoc

```diff
@@ -2,23 +2,31 @@
 * xref:index.adoc[Overview]
 * xref:apikit-release-notes.adoc[Release Notes]
 * xref:apikit-compatibility.adoc[APIkit Compatibility]
+
+* xref:apikit-for-asyncapi.adoc[APIkit for AsyncAPI]
```

Comment:
I assume this link text is hard-coded for a reason?

Reply:
No reason! Oops! 
I removed other unnecessary hardcoded texts.
Thanks!